// let arr = [{ a: 1 }, { a: 2 }, null, { a: 3 }]
// for (let val of arr) {
//   try {
//     setTimeout(() => {
//       console.log(val.a)
//     }, 100);
//   } catch (e) {
//     console.log(e);
//   }
// }
// // 捕获错误
// process.on('uncaughtException', err => {
//   //do something
//   console.log(err)
// })
for (let i = 0; i < 1; i ++) {
  console.log(i)
}